internal/constants/constants.go

```diff
@@ -123,7 +123,7 @@ const (
 )
 
 const (
-	AZURE_AD_PROVIDER_VERSION_CONSTRAINT = ">= 2.53.1"
+	AZURE_AD_PROVIDER_VERSION_CONSTRAINT = ">= 3.0.1"
 	RANDOM_PROVIDER_VERSION_CONSTRAINT   = ">= 3.6.3"
-	AZAPI_PROVIDER_VERSION_CONSTRAINT    = ">= 1.15.0"
+	AZAPI_PROVIDER_VERSION_CONSTRAINT    = ">= 2.0.0-beta"
```

Comment:
Don't know if this constraint will actually work because you can't >= releases with suffixes

Reply:
yes, it failed in acc tests. Reverting back.